src/wallet/components/WalletDropdownFundLink.tsx

```diff
@@ -0,0 +1,76 @@
+import { isValidElement, useMemo } from 'react';
+import { FundWalletSvg } from '../../internal/svg/fundWallet';
+import { cn, pressable, text as themeText } from '../../styles/theme';
+import { version } from '../../version';
+import type { WalletDropdownFundLinkReact } from '../types';
+import { getWindowDimensions } from '../utils/getWindowDimensions';
+
+export function WalletDropdownFundLink({
+  className,
+  icon,
+  rel,
+  popupFeatures,
+  openIn = 'tab',
+  target,
+  text = 'Fund wallet',
```

Comment:
Do we want to offer text override? 

In the BaseNames component we hardcode the text, I feel like we could do the same with the fund link.

Reply:
design wanted a text override